Makefile

```diff
@@ -121,6 +121,15 @@ lint-go-code: $(GOLANGCI_LINT_BIN)
 $(GOLANGCI_LINT_BIN):
 	$(Q)curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./out v1.17.1
 
+.PHONY: courier
+## Validate manifests using operator-courier
+courier:
+	$(Q)python3 -m venv ./out/venv3
+	$(Q)./out/venv3/bin/pip install --upgrade setuptools
+	$(Q)./out/venv3/bin/pip install --upgrade pip
+	$(Q)./out/venv3/bin/pip install operator-courier==2.0.2
+	$(Q)./out/venv3/bin/operator-courier flatten ./manifests ./out/manifests
```

Comment:
The `manifests` directory is already in the flat structure, right?
Can we directly run the `verify` command against this directory?

Reply:
Yes we can, but our `manifests` directory structure has changed so many times in last few days. And if somebody again try to make it according to operator-registry's manifests format this code will break. I already experienced this few time in last week. So its better to keep flatten command here. And anyway its creating flatten dir in out dir. So won't matter much.